content/applications/finance/accounting/bank/loans.rst

```diff
@@ -0,0 +1,72 @@
+:show-content:
+
+================
+Loans management
+================
+
+Odoo lets you manage your loans to have a comprehensive list of all loans undertaken by your company
+in order to maintain a holistic and forecasted view of upcoming due dates (e.g., cash forecast).
+Set up your amortization schedule—or import it—and let Odoo automatically handle monthly interest
+and principal adjustments such that your financial reports are always accurate with minimal effort.
```

Comment:
```suggestion
Odoo's loan management gives a comprehensive list of all loans undertaken by your company in order
to maintain a holistic and forecasted view of upcoming due dates (e.g., cash forecast). Set up
amortization schedules—or import them—and let Odoo automatically handle monthly interest and
principal adjustments so that your financial reports are always accurate with minimal effort.
```